package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Volkswagen detects when your tests are being run in a CI server, and makes them pass.",
   "main": "./.these/aren't/the/source/files/you're/looking/for.js",
   "dependencies": {
+    "chai": "^3.3.0",
```

Comment:
shouln't [try/catch](https://github.com/ignovak/volkswagen/blob/defeat_chai/.these/aren't/the/source/files/you're/looking/for.js#L10-L12) used instead?